README.md

```diff
@@ -1,10 +1,10 @@
 # EncompassRest
-Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+.
+Encompass API Client Library for .NET Framework 4.5+ and .NET Standard 1.1+
```

Comment:
I know this is petty but we lost a period here.

Reply:
hmm.  i thought it looked silly there. ill put it back.